Appsilon/rhino
# Box trailing commas linter

### Changes
* Closes #534
* Adds a `box_trailing_commas_linter()`
* Adds the linter to the `.lintr` template
* Adds documentation

### How to test
1. Unit tests for `box_trailing_commas_linter()` exist
2. Locally install `rhino`, version should be `1.7.0.9002`, initialize a new rhino application then proceed to declare box imports.
3. `rhino::lint_r()` should find and point to all package, module, function imports without a trailing comma.
4. Or, with an existing rhino app, add `box_trailing_commas_linter = rhino::box_trailing_commas_linter()` to `.lintr`.
5. `rhino::lint_r()` should find and point to all package, module, function imports without a trailing comma. 

### Known issues

```
> devtools::document()
ℹ Loading rhino
Warning message:
[linter_box_trailing_commas_linter.R:7] @examples has mismatched braces or quotes 
```

I believe it sees the trailing commas in the example and complains.
